runtime/themes/iceberg-dark.toml

```diff
@@ -0,0 +1,126 @@
+"diff.minus" = { fg = "red" }
+"diff.minus.gutter" = { fg = "red", bg = "linenr_bg" }
+"diff.plus" = { fg = "green" }
+"diff.plus.gutter" = { fg = "green", bg = "linenr_bg" }
+"error" = { fg = "red" }
+"exception" = { fg = "blue" }
+"field" = { fg = "background_fg" }
+"float" = { fg = "purple" }
+"function" = { fg = "pale" }
+"function.macro" = { fg = "green" }
+"hint" = { fg = "comment_fg" }
+"identifier" = { fg = "blue" }
+"info" = { fg = "cyan" }
+"keyword" = { fg = "blue" }
+"keyword.directive" = { fg = "green" }
+"keyword.import" = { fg = "pale" }
+"label" = { fg = "green" }
+"markup.bold" = { modifiers = ["bold"] }
+"markup.heading" = { fg = "blue", modifiers = ["bold"] }
+"markup.italic" = { modifiers = ["italic"] }
+"markup.link" = { fg = "blue", underline = { style = "line" } }
+"markup.link.label" = { fg = "cyan" }
+"markup.link.text" = { fg = "cyan" }
+"markup.link.url" = { underline = { style = "line" } }
+"markup.list" = { fg = "orange", modifiers = ["bold"] }
+"markup.raw" = { fg = "cyan" }
+"markup.raw.inline" = { bg = "black", fg = "blue" }
+"markup.strikethrough" = { modifiers = ["crossed_out"] }
+"method" = { fg = "pale" }
+"namespace" = { fg = "blue" }
+"number" = { fg = "purple" }
+"operator" = { fg = "blue" }
+"parameter" = { fg = "background_fg" }
+"property" = { fg = "background_fg" }
+"punctuation.bracket" = { fg = "background_fg" }
+"punctuation.delimiter" = { fg = "background_fg" }
+"punctuation.special" = { fg = "green" }
+"repeat" = { fg = "blue" }
+"special" = { fg = "green" }
+"string" = { fg = "cyan" }
+"string.escape" = { fg = "green" }
+"string.special" = { fg = "green" }
+"tag" = { fg = "blue" }
+"tag.attribute" = { fg = "purple" }
+"text" = { fg = "background_fg" }
+"type" = { fg = "blue" }
+"ui.background" = { fg = "background_fg", bg = "background_bg" }
+"ui.background.separator" = { fg = "comment_fg" }
+"ui.bufferline.active" = { fg = "pale" }
+"ui.cursor.match" = { fg = "background_fg", bg = "matchparen_bg" }
+"ui.cursor.normal" = { bg = "gray" }
+"ui.cursor.primary" = { modifiers = ["reversed"] }
+"ui.cursor.select" = { bg = "gray" }
+"ui.gutter" = { fg = "linenr_fg", bg = "linenr_bg" }
+"ui.help" = { fg = "background_fg", bg = "cursorlinenr_bg" }
+"ui.linenr" = { fg = "linenr_fg", bg = "linenr_bg" }
+"ui.menu" = { fg = "background_fg", bg = "cursorlinenr_bg" }
+"ui.menu.border" = { fg = "comment_fg" }
+"ui.menu.selected" = { fg = "menusel_fg", bg = "menusel_bg" }
+"ui.popup" = { fg = "background_fg", bg = "cursorlinenr_bg" }
+"ui.popup.info" = { fg = "blue" }
+"ui.selection" = { bg = "sel_bg" }
+"ui.statusline" = { bg = "statusline_bg", fg = "statusline_fg" }
+"ui.statusline.insert" = { fg = "black", bg = "blue" }
+"ui.statusline.select" = { fg = "black", bg = "green" }
+"ui.text.focus" = { fg = "orange" }
+"ui.virtual.indent-guide" = { fg = "linenr_fg" }
+"ui.virtual.jump-label" = {  fg = "orange", modifiers = ["bold"] }
+"ui.virtual.whitespace" = { fg = "sel_bg" }
```

Comment:
You may want to set a default `ui.virtual` key for any future features that use that namespace, for example `"ui.virtual" = { fg = "linenr_fg" }`.

Also you should add a background highlight for rulers (`"ui.virtual.ruler"`). You can enable rulers with `:set rulers [80]` for example to test

Reply:
added in https://github.com/helix-editor/helix/pull/10674/commits/d4ce668093de366e58ec7e6bf9a106e7fc60cd39